<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>831</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(221, 255, 205);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-11</y>
      <width>281</width>
      <height>811</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(25, 99, 90)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Script MT Bold</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string>на вашем счету:</string>
     </property>
    </widget>
    <widget class="QLabel" name="money">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="add">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>330</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 20px;
background-color: rgb(85, 255, 0);
border-radius: 20px;
color: white;</string>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dell">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>440</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 20px;
background-color: rgb(85, 255, 0);
border-radius: 20px;
color: white;</string>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="calendar">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>550</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 20px;
background-color: rgb(85, 255, 0);
border-radius: 20px;
color: white;</string>
     </property>
     <property name="text">
      <string>Календарь</string>
     </property>
    </widget>
    <widget class="QPushButton" name="glav">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>660</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 20px;
background-color: rgb(85, 255, 0);
border-radius: 20px;
color: white;</string>
     </property>
     <property name="text">
      <string>Главная</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>811</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 255, 169)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>261</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yu Gothic UI Semibold</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(254, 255, 169)</string>
     </property>
     <property name="text">
      <string>FAMILY MONEY</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>801</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(254, 255, 169);
color: white</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="search">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>651</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
border-radius: 20px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px;
font: 63 15pt &quot;Yu Gothic UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>найти</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>771</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 13px;
border-radius: 20px;
background-color: rgb(253, 255, 234);</string>
    </property>
    <property name="columnCount">
     <number>5</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>155</number>
    </attribute>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>491</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background-color: rgb(0, 165, 110);
border-radius: 20px;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="dependtable">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>510</y>
      <width>481</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 13px;
border-radius: 20px;
background-color: rgb(253, 255, 234);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="find_button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>440</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px;
background-color: rgb(85, 255, 0);
border-radius: 20px;
color: white;</string>
    </property>
    <property name="text">
     <string>Найти</string>
    </property>
   </widget>
   <zorder>frame_2</zorder>
   <zorder>frame</zorder>
   <zorder>search</zorder>
   <zorder>label_4</zorder>
   <zorder>tableWidget</zorder>
   <zorder>calendarWidget</zorder>
   <zorder>dependtable</zorder>
   <zorder>find_button</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1085</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
